src/modules/site-v2/base/views/api/gene.py

```diff
@@ -25,16 +26,18 @@ def api_search_homologs(query=""):
 def api_search_genes(query=""):
   query = request.args.get('query') or query
   query = str(query).lower()
-  return search_genes(query)
-
-
-@api_gene_bp.route('/search/<string:query>')
-@cache.memoize(60*60)
-@jsonify_request
-def api_search_combined(query=""):
-  query = request.args.get('query') or query
-  query = str(query).lower()
-  return (search_genes(query) + search_homologs(query))[0:10]
+  species = request.args.get('species') or None
```

Comment:
The `#get() + or` can be combined: the 2nd argument to #get(first, second) is the default value. Eg: 
```
a = {}
b = a.get('foo', 'bar')
b == 'bar' # True
```

Reply:
Fixed in 9e2ecf8e2deb2a16724920837796f8f61ccd0b7a